packages/wallet/test/services/util/ProviderTrackerSubject.test.ts

```diff
@@ -47,6 +47,9 @@ describe('ProviderTrackerSubject', () => {
     });
   });
 
+  // Review: this is a list of tests needed to test all paths.
```

Comment:
Happy for us to merge with this as a todo, and we can decide to drop completely or spent the effort when possible